libs/agno/agno/agent/agent.py

```diff
@@ -2187,8 +2267,8 @@ async def _acontinue_run(
         """
 
         # Resolving here for async requirement
-        if self.dependencies is not None:
-            await self._aresolve_run_dependencies()
+        if dependencies is not None:
```

Comment:
Here you are ignoring the `self.dependencies`? It should work the same as on run/arun

Reply:
Ah ok yeah, so it is resolved higher up